src/libs/SearchUIUtils.ts

```diff
@@ -445,6 +445,7 @@ function getAction(data: OnyxTypes.SearchResults['data'], key: string): SearchTr
 function getTaskSections(data: OnyxTypes.SearchResults['data']): TaskListItemType[] {
     return Object.keys(data)
         .filter(isReportEntry)
+        .filter((key) => isTaskListItemType(data[key] as SearchListItem))
```

Comment:
Please add a comment here in the code to explain why these are being filtered out.